src/main/java/dev/jbang/source/sources/GroovySource.java

```diff
@@ -40,14 +40,15 @@ protected List<String> getNativeOptions() {
 	protected List<String> getRuntimeOptions() {
 		List<String> gopts = Collections.singletonList("-Dgroovy.grape.enable=false");
 		List<String> opts = tagReader.collectOptions("JAVA_OPTIONS", "RUNTIME_OPTIONS");
+		opts.remove("-g");
```

Comment:
This is definitely not the way to handle this :-)

But I'm pretty sure this was already fixed by #1657

Reply:
yeah - this should not be needed.

Reply:
Done, and delete this line.